docs/core/diagnostics/debug-memory-leak.md

```diff
@@ -2,7 +2,7 @@
 title: Debug a memory leak tutorial
 description: Learn how to debug a memory leak in .NET Core.
 ms.topic: tutorial
-ms.date: 12/17/2019
+ms.date: 04/20/2020
```

Comment:
Does this really warrant a date update?

Reply:
I can revert if you'd like, typically I read through the article if I'm making updates and verify that everything looks accurate. So it was kind of like a freshness pass. Let me know if you want me to change it back.